pkgs/development/tools/build-managers/buildbot/worker.nix

```diff
@@ -1,18 +1,26 @@
-{ stdenv, fetchurl, pythonPackages }:
+{ stdenv, fetchurl, pythonPackages, gnused, coreutils }:
 
 pythonPackages.buildPythonApplication (rec {
   name = "${pname}-${version}";
   pname = "buildbot-worker";
-  version = "0.9.0.post1";
+  version = "0.9.3";
 
   src = fetchurl {
     url = "mirror://pypi/b/${pname}/${name}.tar.gz";
-    sha256 = "1f8ij3y62r9z7qv92x21rg9h9whhakkwv59rgniq09j64ggjz8lx";
+    sha256 = "176kp04g4c7gj15f73wppraqrirbfclyx214gcz966019niikcsp";
   };
 
   buildInputs = with pythonPackages; [ setuptoolsTrial mock ];
   propagatedBuildInputs = with pythonPackages; [ twisted future ];
 
+  postPatch = ''
+    ${gnused}/bin/sed -i 's|/usr/bin/tail|${coreutils}/bin/tail|' buildbot_worker/scripts/logwatcher.py
+  '';
+
+  postFixup = ''
+    makeWrapper $out/bin/.buildbot-worker-wrapped $out/bin/buildbot-worker --set PYTHONPATH "$PYTHONPATH"
```

Comment:
that appears to be so.  if you look at the changes in the bb master expression, you can see the wrapper issue there as well:

`-    mv -v $out/bin/buildbot $out/bin/.wrapped-buildbot`
`-    echo "#!/bin/sh" > $out/bin/buildbot`
`-    echo "export PYTHONPATH=$PYTHONPATH" >> $out/bin/buildbot`
`-    echo "exec $out/bin/.wrapped-buildbot \"\$@\"" >> $out/bin/buildbot`
`-    chmod -c 555 $out/bin/buildbot`
`+    makeWrapper $out/bin/.buildbot-wrapped $out/bin/buildbot --set PYTHONPATH "$PYTHONPATH"`

it looks like without this wrapper in worker, worker was working on centos, but not nixos.